windowspkg/linkerd2-cli/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,14 @@
+﻿$ErrorActionPreference = 'Stop';
+$toolsPath = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+
+$packageArgs = @{
+
+  packageName    = 'linkerd'
+  fileFullPath   = "$toolsPath\linkerd.exe"
+  url            = 'https://github.com/linkerd/linkerd2/releases/download/stable-2.6.1/linkerd2-cli-stable-2.6.1-windows.exe'
```

Comment:
It looks like stable-2.6.1 is hardcoded here.  Any way to make this dynamic instead?

Could we require the user specifies what version they want?
Could we automatically detect the latest version?

How does Chocolatey handle package versioning?

Reply:
Hi there! Thank you for reviewing the PR :) Yes chocolatey does automatically update this, more is explained [here](https://github.com/majkinetor/au/blob/master/README.md) and as mentioned this will be worked on once the things like `LICENSE.txt`, `VERIFICATION.txt` and the `.nuspec` files are correct (just to make things simpler to review). 

To also answer your previous comment, this package is meant for making installation of linkerd2-cli to windows easier, as mentioned in #3063. The package does not have to live in the linkerd repo (another reason why this is a draft PR), the usual way is to push the chocolatey package as mentioned [here](https://chocolatey.org/docs/create-packages#push-your-package), another add on can also be to add this to the [community packages repo](https://github.com/chocolatey-community/chocolatey-coreteampackages) for chocolatey (there's a Kubernetes cli package in the `automatic` folder here too!). 

To clarify, the goal is to make linkerd2-cli easier to install in windows (in #3063), it does not have to live in the repo, the workflow would simply involve running `choco install linkerd2-cli` once this is pushed to where it will live (either on chocolatey community feed or an external feed maintained by linkerd).

Reply:
Thanks for the clarification, @drholmie, that's very helpful.

I think the biggest open question is, who should be responsible for maintaining the package.  If this is to be the responsibility of the core Linkerd team, then we should definitely add the appropriate files to the Linkerd2 repo and we should make publishing the Chocolatey package a part of the Linkerd2 release process.

On the other hand, my understanding is that anyone could maintain their own Chocolatey package of the Linkerd CLI without any direct involvement from the Linkerd project.  Is that correct?

What do you think is the best way to proceed?

Reply:
Yup, anyone can maintain their own Chocolatey package, although that makes it an unofficial one. There's also the issue with having legal rights to distribute said software, this PR wouldn't violate that either even if it is unofficial as explained [here](https://chocolatey.org/docs/legal). 

However as mentioned in the issue (#3063) and previous discussions on slack, this should ideally be an official package. This makes the PR more complicated, another reason why this is a draft PR so that if necessary it can incorporate those changes as well. 

As far as I can tell, maintaining just the Chocolatey package shouldn't be hard due to the auto update feature. Including this in the release process, CI/CD and so on, I can't really judge as I don't have any experience in it. Since Linkerd's team is going to be the one who maintains it (if it's official), it's better if Linkerd decides how to proceed based on the costs and benefits of making this an official package.

Reply:
It sounds like the next steps would be to add detailed instructions for how to update the Chocolatey package during a release of Linkerd.  That will help us decide if it's something that we want to take on.  Thanks!

Reply:
Alright, will start working on adding Chocolatey's auto update feature then. Thanks again for reviewing the PR!